types.d.ts

```diff
@@ -13,8 +13,10 @@ declare module "hot-shots" {
     isChild?: boolean;
     maxBufferSize?: number;
     mock?: boolean;
+    path?: string;
     port?: number;
     prefix?: string;
+    protocol?: string;
```

Comment:
Shouldn't this be constrained to `'tcp' | 'udp' | 'unix_dgram'`?

Also, this would fix #101.